client/src/models/trees.js

```diff
@@ -84,14 +84,40 @@ const trees = {
         })
       })
     },
+
```

Comment:
Separation of getting tree count for the sake of not having to fetch trees with it.